fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -101,6 +101,34 @@ private void addLoadJob(LoadJob loadJob) {
         loadJobScheduler.submitJob(loadJob);
     }
 
+    public void cancelLoadJob(CancelLoadStmt stmt) throws DdlException {
+        Database db = Catalog.getInstance().getDb(stmt.getDbName());
+        if (db == null) {
+            throw new DdlException("Db does not exist. name: " + stmt.getDbName());
+        }
+
+        LoadJob loadJob = null;
+        readLock();
+        try {
+            if (!dbIdToLabelToLoadJobs.containsKey(db.getId())) {
+                throw new DdlException("Load job does not exist");
+            }
+            if (!dbIdToLabelToLoadJobs.get(db.getId()).containsKey(stmt.getLabel())) {
+                throw new DdlException("Load job does not exist");
+            }
+            List<LoadJob> loadJobList = dbIdToLabelToLoadJobs.get(db.getId()).get(stmt.getLabel());
```

Comment:
you execute `containKey` or `get` multiple times. please use a local variable to avoid that